web/js/containers/gif.js

```diff
@@ -266,15 +266,28 @@ class GIF extends Component {
     if (top + height > screenHeight) {
```

Comment:
```suggestion
   if (left + width > screenWidth && x - 20 - width < 0) {
      left = left - width;
    }
  if (top + height > screenHeight) {
```
I think preventing the modal from going outside the viewport is a good call for this release, Having the modal be draggable doesn't seem as urgent.

Reply:
Sounds good.  I added a little bit more handling for cases when the crop area was too wide to leave room for the modal.   Now the modal will move to the top/bottom of the crop area instead of overlap and only overlaps if there's absolutely nowhere else to go.

Reply:
Nice!